### PR TITLE
add specific policies to ambient getting started guide

### PR DESCRIPTION
**Pull request by craigbox**
Make more obvious that you need L4 and L7 policies.

### CODE REVIEW

File: content/en/docs/ambient/getting-started/enforce-auth-policies/index.md [hunk 1]
```diff
@@ -96,8 +96,31 @@ EOF
...
 Note the `targetRefs` field is used to specify the target service for the authorization policy of a waypoint proxy. The rules section is similar as before, but this time you added the `to` section to specify the operation that is allowed.
 
+Remember that our L4 policy instructed the ztunnel to only allow connections from the gateway? We now need to update it to also allow connections from the waypoint.
+
+{{< text syntax=bash snip_id=update_l4_policy >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: productpage-viewer-ztunnel
```
Comment (ilrudie) [inline content/en/docs/ambient/getting-started/enforce-auth-policies/index.md:106]:
```suggestion
  name: productpage-ztunnel
```

**Review by ilrudie — APPROVED** [review]
LGTM, I like the level of detail here and I think it's more illustrative of how the system works.

**Review by keithmattix — APPROVED** [review]
Thanks Craig!

File: content/en/docs/ambient/getting-started/enforce-auth-policies/index.md (hunk 1 above)
Comment (craigbox) [reply content/en/docs/ambient/getting-started/enforce-auth-policies/index.md:106]: good catch, thanks!

**Review by kfaseela — APPROVED** [review]

**Pull request merged**